hack/update-deps.sh

```diff
@@ -31,8 +31,8 @@ cd "${ROOT_DIR}"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/test-infra@${VERSION}"
```

Comment:
Change `VERSION` var above instead?

Reply:
I don't have a preference, this is what I've seen in the past. It doesn't make a difference in this repo, but in others where we import both pkg and eventing, we can't use `VERSION` for all of them (as they are out of sync for 1 week prior to releases).